tests/test_modeling_tf_common.py

```diff
@@ -88,20 +91,28 @@ def _prepare_for_class(self, inputs_dict, model_class, return_labels=False):
 
         if return_labels:
             if model_class in TF_MODEL_FOR_MULTIPLE_CHOICE_MAPPING.values():
-                inputs_dict["labels"] = tf.ones(self.model_tester.batch_size)
+                inputs_dict["labels"] = tf.ones(self.model_tester.batch_size, dtype=tf.int32)
```

Comment:
Like Patrick's comment above, a lot of the tests have identical code in their conditions.